eksupgrade/models/eks.py

```diff
@@ -97,23 +401,26 @@ class ClusterAddon(EksResource):
     product_id: str = ""
     product_url: str = ""
     configuration_values: str = ""
-    cluster: Cluster = field(default_factory=lambda: Cluster(arn=""))
+    cluster: Cluster = field(default_factory=lambda: Cluster(arn="", version="1.24"))
```

Comment:
is this default version necessary/required? is there any possibility of someone updating a cluster and this is used instead of the control plane's current version?

Reply:
Will adjust the init workflow for this to eliminate the requirement.  It shouldn't ever occur within the scripted workflow, but (to your point) could absolutely cause version confusion if used programmatically (and in other custom workflows).
I will modify the version parsing so this shortcut isn't required in my follow-up PR with parallel execs.